pkg/kubernetes/scripts/virtual-machines/index.jsx

```diff
@@ -49,9 +49,9 @@ function addKubeLoaderListener ($scope, kubeLoader, kubeSelect) {
     kubeLoader.watch('VirtualMachine', $scope);
 }
 
-const VmsPlugin = () => (
-    <Provider store={reduxStore} >
-        <VmsListing />
+const VmsPlugin = ({ kubeMethods, kubeLoader }) => (
+    <Provider store={reduxStore}>
+        <VmsListing kubeLoader={kubeLoader} kubeMethods={kubeMethods}/>
```

Comment:
Maybe passing `kubeLoader` and `kubeMethods` here and ant line 52 is unnecessary.